simapp/app.go

```diff
@@ -76,21 +76,23 @@ func MakeCodec() *codec.Codec {
 	return cdc
 }
 
-// Extended ABCI application
+// App extends an ABCI application, but with most of its parameters exported.
+// They are exported for convenience in creating helper functions, as object
+// capabilities aren't needed for testing.
 type SimApp struct {
 	*bam.BaseApp
-	cdc *codec.Codec
+	Cdc *codec.Codec
 
 	invCheckPeriod uint
 
 	// keys to access the substores
-	keys  map[string]*sdk.KVStoreKey
+	Keys  map[string]*sdk.KVStoreKey
 	tkeys map[string]*sdk.TransientStoreKey
 
 	// keepers
-	accountKeeper  auth.AccountKeeper
-	bankKeeper     bank.Keeper
-	supplyKeeper   supply.Keeper
+	AccountKeeper  auth.AccountKeeper
```

Comment:
Is there a reason certain keepers are public and others are kept private?

Reply:
no reason, will update the rest